tests/datasets/test_cms_mangrove_canopy.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can you create a `tests/data/cms_mangrove_canopy/data.py` script that generates the fake test data above? This helps us to easily recreate the test data if we need to increase the image sizes. Someday we're planning on generating data on the fly instead of storing fake data in git. See other datasets with a `data.py` file for examples.